atlas-db-controller/controller.go

```diff
@@ -628,7 +628,7 @@ func (c *Controller) syncDatabaseSecret(key string, db *atlas.Database, dbserver
 		for _, user := range db.Spec.Users {
 			passwd := user.Password
 			if user.Role == "admin" {
-				if passwd == "" {
+				if user.PasswordFrom != nil {
```

Comment:
So, if the resource has both "Password" and a valid "PasswordFrom", we will take "PasswordFrom" as the one to use. Is that consistent with how we do the other, similar fields (git/gitFrom)?

Reply:
Our intention is to get the password from the passwordFrom and put in to the db object defined in code as password(line number:640). Then it can use for  create database users defined in postgres plugin package. so when calling p.syncdatabase(line no:594) passing the db object with user password instead of passwordfrom.

Our assumption is when next cycle comes and it checks "if passwd =="" " , this condition failed and it wont go into that condition  because the previous passwd exists.
That is why given  "if user.PasswordFrom != nil" .
correct us if we are wrong.